client/src/components/ratings-reviews/RatingSummary.jsx

```diff
@@ -0,0 +1,53 @@
+/* eslint-disable nonblock-statement-body-position */
+/* eslint-disable no-else-return */
+/* eslint-disable max-len */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable react/prefer-stateless-function */
+import React from 'react';
+
+class RatingSummary extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    if (!this.props.summary['5']) {
```

Comment:
It looks like you're trying to copy the ratings data over to summary? Could you just do `let summary = this.props.ratings` ?

Reply:
Thanks for pointing this out